arch/arm/src/stm32f7/hardware/stm32_sdmmc.h

```diff
@@ -46,7 +46,7 @@
 #define STM32_SDMMC_ARG_OFFSET                0x0008 /* SDMMC argument register */
 #define STM32_SDMMC_CMD_OFFSET                0x000c /* SDMMC command register */
 #define STM32_SDMMC_RESPCMD_OFFSET            0x0010 /* SDMMC command response register */
-#define STM32_SDMMC_RESP_OFFSET(n)            (0x0010+4*(n))
+#define STM32_SDMMC_RESP_OFFSET(n)            (0x0010 + 4 * (n))
```

Comment:
sqaush with the previous patch

Reply:
I can do that. However, the first commit is a clear merge only, and this one is a style change. Up to you.